### ISSUE TITLE
add more `accessibility.verbosity` settings / hints 

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#180470]
With the initial small set that we allow configuring in #180365, we should find other components that would benefit from:

1. a setting like this
2. an `aria-label` with helpful information and then a setting like this

_Originally posted by @meganrogge in https://github.com/microsoft/vscode/issues/172465#issuecomment-1515470745_

**Comment by amunger** [comment]
I noticed that the notebook editor has an aria alert when certain parts of a cell are focused with some navigation hints. I haven't actually seen it triggered much when playing around with it, but it seems potentially noisy and could benefit from this setting - https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts#L2239

**Comment by mjbvz** [comment]
Should also potentially be added to the public api on the `AccessibilityInformation` type

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by meganrogge** [comment]
I'll continue to think about this, but considering this finished for now

**Comment by meganrogge** [comment]
Verify that the accessibility verbosity setting description makes sense and that it works when you have a screen reader on for `accessibility.verbosity.notebook` and `accessibility.verbosity.keybindingsEditor`

for notebooks, you should see `press escape to focus the cell container`... or `press enter to focus the inner...` depending on where you click.

for the keybindings editor, when you focus a row, you should see `use space or enter to change the keybinding`

**Comment by meganrogge** [comment]
@rebornix can you pls provide more specific steps for the notebook case as I found it hard to trigger as did @amunger ?

**Comment by joyceerhl** [comment]
If I have a markdown cell in preview mode and hit enter, I hear 'press escape to focus the cell container'. If I then press escape to put the cell back into preview mode, I then hear 'press enter to focus the inner markdown editor`. These instructions are not announced if I have `accessibility.verbosity.notebook` disabled ✅.

When I click into a notebook cell editor I hear 'the editor is not accessible at this time', which I don't really understand, is that expected?

**Comment by meganrogge** [comment]
> When I click into a notebook cell editor I hear 'the editor is not accessible at this time', which I don't really understand, is that expected?

That sounds like a bug, but unrelated to these changes. @rebornix might have more info

**Comment by joyceerhl** [comment]
I'll file an issue
